src/g_syscalls.c

```diff
@@ -406,9 +406,9 @@ intptr_t trap_Map_Extension(const char *ext_name, intptr_t mapto)
 	return syscall(G_Map_Extension, (intptr_t) ext_name, mapto);
 }
 
-intptr_t trap_AddBot(const char *name, intptr_t bottomcolor, intptr_t topcolor, const char *skin)
+intptr_t trap_AddBot(const char *name, intptr_t bottomcolor, intptr_t topcolor, const char *skin, intptr_t skill)
```

Comment:
To [quote](https://discord.com/channels/166866762787192833/179895022366228481/1271384099558391898) @qqshka 
> Not sure we really need bot skill as a ping, probably can just set user info key for bot, skill or something

Here's a sample of how it works with team/topcolor/bottomcolor:
https://github.com/QW-Group/ktx/blob/master/src/vote.c#L715-L717

Although mvdsv has this comment:
```c
// FIXME: Why PR2_UserInfoChanged is not called here? Like for normal players.
//        Why we need this special handling in the first place?
```
So perhaps the regular userinfo, check with someone who knows more.

Reply:
i think the purpose of showing bot skill in ping column is so you can see from the endgame screenshot that the proper skill was used to achieve your score. I suppose it can be removed entirely if it's not wanted.